ParabolInc/parabol
# feat(prompt-to-join): Add teammate mutation

- Fixes #7803
- In this PR I also remove unnecessary enterprise restriction on sending join request 
- Added segment events

https://user-images.githubusercontent.com/466991/236413243-0bd08389-183a-451c-bd06-7426d6269315.mov

**How to test:** 
- Follow previous PR testing steps to see add teammate UI (#8086)
- Test that adding a user works correctly
- Check that segment events sent correctly
